snippets/olmv1-known-issue-service-accounts.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/olmv1-installing-an-operator.adoc
+// * release_notes/ocp-4-17-release-notes.adoc (enterprise-4.17 branch only)
+
+:_mod-docs-content-type: SNIPPET
+
+There is a known issue in {olmv1}. If you do not assign the correct role-based access controls (RBAC) to an extension's service account, {olmv1} gets stuck and reconciliation stops.
+
+Currently, {olmv1} does not have tools to help extension administrators find the correct RBAC for a service account.
+
+Because {olmv1} is a Technology Preview feature and must not be used on production clusters, you can avoid this issue by using the more permissive RBAC included in the documentation.
+
+This RBAC is intended for testing purposes only. Do not use it on production clusters. (link:https://issues.redhat.com/browse/OCPBUGS-38153[*OCPBUGS-38153*]) and (link:https://issues.redhat.com/browse/OCPBUGS-38155[*OPCBUGS-38155*])
```

Comment:
@michaelryanpeter I do not think https://issues.redhat.com/browse/OCPBUGS-38155 is related to RBAC because when I tested it, I use cluster-amdin permssion
```yaml
- apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRole
  metadata:
    name: ns-75493-sa75493
  rules:
  - apiGroups:
    - "*"
    resources:
    - "*"
    verbs:
    - "*"
```
so, please remove the linked bug.

Reply:
Addressed here: https://github.com/openshift/openshift-docs/pull/82245/commits/dcc24db551944706efa989b0b5fd9d37ebce362f#diff-4a8dbc4b978a576dd34a58723ce7ea341d838ed8cf062a588a0935d1e0d37526R14